cognite_toolkit/_cdf.py

```diff
@@ -1128,6 +1166,82 @@ def dump_asset_cmd(
             )
         )
 
+    @dump_app.command("timeseries")
```

Comment:
I think this deserves its own feature flag, and not be with the same one as assets. Can you add another and add a if-block here?

Reply:
Agree with separate FEATURE.

Missed before and found now `setup_asset_loader.py` and it is above my head to implement it for timeseries?! Which `_modify` steps to implement?

Reply:
As suggested in the main comment, I think you should skip the modification of the loader. And instead just focus on the new dump functionality. 

This means that the feature this PR should provide is ONLY dump of timeseries, but not ability to write timeseries from parquet and csv to CDF. Lets instead consider that in 0.3.0a1.